src/collections/_documentation/error-reporting/configuration/draining.md

```diff
@@ -3,15 +3,9 @@ title: 'Shutdown and Draining'
 sidebar_order: 1
 ---
 
-Most SDKs use a background queue to send out events.  Because the queue sends asynchronously in the background
-it means that some events might be lost if the application shuts down unexpectedly.  To prevent this all SDKs
-provide mechanisms to cope with this.
-
-Typically SDKs provide two ways to shut down: a controlled shutdown where the system will wait up to about two
-seconds to flush out events (configurable) and an uncontrolled shutdown (also referred to as "killing" the
-client).
```

Comment:
This doesn't sound useful to me. It is a generic statement, and unclear whether it applies to the SDK the user is interested in.

Each language/platform from the dropdown includes the relevant mechanisms that exist in that language/platform.